src/main/java/io/botcrafting/botcraft/core/handler/processors/MessageProcessor.java

```diff
@@ -0,0 +1,10 @@
+package io.botcrafting.botcraft.core.handler.processors;
```

Comment:
as mentioned in a comment down below processors looks generic, let's maybe change to repliers